eZ/Bundle/EzPublishCoreBundle/Tests/ApiLoader/RepositoryConfigurationProviderTest.php

```diff
@@ -93,6 +93,26 @@ public function testGetRepositoryConfigUndefinedRepository()
         $provider->getRepositoryConfig();
     }
 
+    public function testGetDefaultAndCurrentRepositoryAlias(): void
+    {
+        $configResolver = $this->getConfigResolverMock();
+
+        $repositories = [
+            'first' => [
+                'engine' => 'foo',
+            ],
+            'second' => [
+                'engine' => 'bar',
+            ],
+        ];
+
+        $provider = new RepositoryConfigurationProvider($configResolver, $repositories);
+        $provider->getRepositoryConfig();
+
+        $this->assertSame('first', $provider->getDefaultRepositoryAlias());
+        $this->assertSame('first', $provider->getCurrentRepositoryAlias());
```

Comment:
Well, this is unit tests which means that each code unit should be tested separately. I recommend to split getDefaultRepositoryAlias() and getCurrentRepositoryAlias() tests.

Reply:
Resolved in: https://github.com/ezsystems/ezpublish-kernel/pull/3085/commits/c6f4d0939bcaec353c790d40a8dd30cf9140cc22